app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AgentAssuranceConnector.scala

```diff
@@ -35,6 +37,26 @@ class AgentAssuranceConnector @Inject()(@Named("agent-assurance-baseUrl") baseUr
     }
   }
 
+  def getActiveCesaRelationship(url: String)(implicit hc: HeaderCarrier): Future[Boolean] = {
+    http.GET[HttpResponse](baseUrl + url).map(
+      response => response.status == 200)
```

Comment:
I believe it is already done as part of `HttpErrorFunctions.handleResponse` invoked from `JsonHttpReads`, so we don't need to check if status is OK